src/pretalx/orga/forms/event.py

```diff
@@ -37,19 +37,19 @@ class EventForm(ReadOnlyFlag, I18nModelForm):
     )
     logo = ImageField(
         required=False,
-        label=_("Header image"),
+        label="",
         help_text=_(
-            "If you provide a header image, it will be displayed instead of your event's color and/or header pattern "
-            "on top of all event pages. It will be center-aligned, so when the window shrinks, the center parts will "
-            "continue to be displayed, and not stretched."
+            "If you provide a logo image, we will by default not show your event's name and date in the page header. "
+            "We will show your logo in its full size if possible, scaled down to the full header width."
```

Comment:
"scaled down to the full header width" isn't really accurate, as the 200px height will be the limiting factor for most events, I think. I'll fix this in a follow-up commit.